tools/topology/topology2/CMakeLists.txt

```diff
@@ -17,4 +17,8 @@ else()
 	endif()
 endif()
 
-add_subdirectory(cavs)
+add_custom_target(topology2_cavs)
+add_custom_target(topology2_ace)
+add_dependencies(topologies2 topology2_cavs topology2_ace)
```

Comment:
Please move these new targets to their corresponding subdirectories to keep them self-contained.

Reply:
fixed

Reply:
I would prefer each child to add itself to the parent but no big deal.